custom-steps/ffmpeg/post-build.ps1

```diff
@@ -0,0 +1,92 @@
+param (
+    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+)
+
+. "$PSScriptRoot\..\..\util.ps1"
+
+$IsOnMacOS = Check-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "Not on Mac OS.  Exiting..."
+    exit
+}
+
+Write-Host "Running post-build script..."
+Push-Location "$BuildArtifactsPath/lib"
+
+# Enumerate files
+$main_files = @()
+$files_with_versions = @()
+$dylib_files = Get-ChildItem -Path . -Filter "*.dylib"
+foreach ($dylib_file in $dylib_files) {
+    # Check if the file name contains more than one dot
+    if (-not ($dylib_file.Name -match '\..*\..*')) {
+        $main_files += $dylib_file.Name
+    }
+    else {
+        $files_with_versions += $dylib_file.Name
+    }
+}
+
+Write-Host ""
+Write-Host ""
+Write-Host "Dependencies before changes..."
+foreach($main_file in $main_files) {
+    Write-Host "> $main_file"
+    Invoke-Expression "otool -L '$main_file' | grep '@rpath'"
+}
+
+Write-Host ""
+Write-Host ""
+Write-Host "Updating dependencies..."
```

Comment:
Suggestion: "Updating paths to dynamic dependencies" since we're not actually changing the dependencies themselves.

Reply:
Fixed: https://github.com/TechSmith/ThirdParty-Packages-vcpkg/pull/1/commits/ae98c7b6d10347d259de0fd359ad454f1aec527a